pkgs/development/tools/rust/bindgen/default.nix

```diff
@@ -56,7 +56,7 @@ rustPlatform.buildRustPackage rec {
       As with most compiler related software, this will only work
       inside a nix-shell with the required libraries as buildInputs.
     '';
-    homepage = https://github.com/rust-lang-nursery/rust-bindgen;
+    homepage = https://github.com/rust-lang/rust-bindgen;
```

Comment:
No comment in the commit message why this was changed?

Reply:
Thanks for the review, `rust-bindgen` is moved to `rust-lang` from `rust-lang-nursery`.

Reply:
> in the commit message

:smile:

Reply:
The commit message has been updated.
Thank you!